### PR TITLE
Add TI CC32XX to Github CI

### PR DESCRIPTION
**Pull request by abiradarti**
Added TI CC32xx github workflow to build the lock-app example and report size stats.

### CODE REVIEW

File: examples/lock-app/cc32xx/main/CHIPDeviceManager.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ *    @file
+ *      This file implements the CHIP Device Interface that is used by
+ *      applications to interact with the CHIP stack
+ *
+ */
+
+#include <stdlib.h>
+
+#include "CHIPDeviceManager.h"
+// #include <app/ConcreteAttributePath.h>
```
Comment (andy31415) [inline examples/lock-app/cc32xx/main/CHIPDeviceManager.cpp:28]: consider not checking in commented out code

File: examples/lock-app/cc32xx/main/CHIPDeviceManager.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+{
+    CHIPDeviceManagerCallbacks * cb = reinterpret_cast<CHIPDeviceManagerCallbacks *>(arg);
+    if (cb != nullptr)
+    {
+        cb->DeviceEventCallback(event, reinterpret_cast<intptr_t>(cb));
+    }
+}
+
+/**
+ *
```
Comment (andy31415) [inline examples/lock-app/cc32xx/main/CHIPDeviceManager.cpp:53]: Why the empty comment?

File: examples/lock-app/cc32xx/main/include/CHIPDeviceManager.h [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    //  *
+    //  * @param endpoint           endpoint id
+    //  * @param clusterID          cluster id
+    //  * @param attributeId        attribute id that was changed
+    //  * @param manufacturerCode   manufacturer code
+    //  * @param type               attribute type
+    //  * @param size               size of the attribute
+    //  * @param value              pointer to the new value
+    //  */
+    // virtual void PostAttributeChangeCallback(chip::EndpointId endpoint, chip::ClusterId clusterId, chip::AttributeId attributeId,
```
Comment (andy31415) [inline examples/lock-app/cc32xx/main/include/CHIPDeviceManager.h:72]: Delete commented out code?

**Comment by abiradarti** [comment]
@andy31415 Do you have any hints on how to resolve the errors from the jobs that are failing? They all point to the all_targets_linux_x64.txt file saying there is a difference between actual and expected output, and at first I thought all I needed to do was add the cc32xx-lock to the list, but they are still failing so I am not sure what to do.

**Review by andy31415 — APPROVED** [review]

File: .github/workflows/examples-cc32xx.yaml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    pull_request:
+
+concurrency:
+    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    cc32xx:
+        name: cc32xx
+        timeout-minutes: 60
```
Comment (bzbarsky-apple) [inline .github/workflows/examples-cc32xx.yaml:28]: This should probably be longer if just the "Build examples" step can go 60 minutes...

**Review by bzbarsky-apple — APPROVED** [review]

File: .github/workflows/examples-cc32xx.yaml (hunk 4 above)
Comment (abiradarti) [reply .github/workflows/examples-cc32xx.yaml]: increased to 100

**Pull request merged**